internal/cli/kraft/cloud/cloud.go

```diff
@@ -81,6 +84,15 @@ func NewCmd() *cobra.Command {
 	cmd.AddGroup(&cobra.Group{ID: "kraftcloud-instance", Title: "INSTANCE COMMANDS"})
 	cmd.AddCommand(instance.NewCmd())
 
+	cmd.AddGroup(&cobra.Group{ID: "kraftcloud-vol", Title: "VOLUME COMMANDS"})
+	cmd.AddCommand(volume.NewCmd())
+
+	cmd.AddGroup(&cobra.Group{ID: "kraftcloud-svc", Title: "SERVICE COMMANDS"})
+	cmd.AddCommand(service.NewCmd())
+
+	cmd.AddGroup(&cobra.Group{ID: "kraftcloud-quota", Title: "QUOTA COMMANDS"})
```

Comment:
Since quotas does not have subsequent subcommands, this group can be removed:
```diff
- cmd.AddGroup(&cobra.Group{ID: "kraftcloud-quota", Title: "QUOTA COMMANDS"})
```
Move the `AddCommand` to be under the one for deploy.